rdkit/Chem/Subshape/SubshapeAligner.py

```diff
@@ -33,10 +31,6 @@ def _getAllTriangles(pts, orderedTraversal=False):
     for j in range(jStart, len(pts)):
       if j == i:
         continue
-      if orderedTraversal:
```

Comment:
This dead code might be indicative of a real bug.

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review. I will make a doc of this for further usage